account_cutoff_prepaid/models/account_cutoff.py

```diff
@@ -97,7 +95,8 @@ def _prepare_prepaid_lines(self, aml, mapping):
                 prepaid_days = total_days
             else:
                 prepaid_days = (end_date_dt - cutoff_date_dt).days
-        assert total_days > 0, "Should never happen. Total days should always be > 0"
+        if total_days > 0:
+            raise ValidationError(_("Total days should always be > 0"))
```

Comment:
Arf, that's a big mistake. It seems you didn't test your v13 migration in real life !
I'm preparing a PR to fix all this and get the improvements from v12 too.